README.md

```diff
@@ -550,9 +550,7 @@ If a malicious site tries to embed the first party on their own site, then the m
     <br><br>
 </figure></center>
 
-It's important to note that `Partitioned` does not offer all of the same protections as `SameSite=Lax/Strict`.
-For example, consider the case when `3p.com` is compromised by a malicious actor and is still embedded on `1p.com`.
-In that case, the attacker could embed `1p.com` into `3p.com`'s frame when `1p.com` is the top-level site, and the attacker would have access to `1p.com`'s `Partitioned` cookies.
+It's important to note that `Partitioned` does not offer all of the same protections as `SameSite=Lax/Strict`. For example partitioned cookies may be sent in cross-site requests from iframes which is prohibitied by SameSite-Lax/Strict.
```

Comment:
Given that you have to explicitly mark partitioned cookies as SameSite=None it's probably fine to delete this entire paragraph?